content/en/docs/concepts/configuration/manage-resources-containers.md

```diff
@@ -389,73 +288,96 @@ run on. Each node has a maximum amount of local ephemeral storage it can provide
 
 The scheduler ensures that the sum of the resource requests of the scheduled Containers is less than the capacity of the node.
 
-### How Pods with ephemeral-storage limits run
+### Ephemeral storage consumption management {#resource-emphemeralstorage-consumption}
 
-For container-level isolation, if a Container's writable layer and logs usage exceeds its storage limit, the Pod will be evicted. For pod-level isolation, if the sum of the local ephemeral storage usage from all containers and also the Pod's emptyDir volumes exceeds the limit, the Pod will be evicted.
+If the kubelet is managing local ephemeral storage, then the kubelet
+measures storage use in:
+- `emptyDir` volumes
```

Comment:
I believe we only track usage of boot-disk backed empydirs, not memory? cc @jingxu97 @dashpole

Reply:
yes, that is correct.

Reply:
How's this?

Reply:
@dashpole can you help review correctness on what happens when different layers are not on the root disk?

Reply:
Memory usage is accounted using cgroups, and is charged to container processes.  The kubelet's empty-dir monitoring functions are only run on disk-backed empty-dir volumes.

Reply:
The rendered page finishes the sentence with “where these are backed by the system root filesystem.” Is that clear enough?

Reply:
gstm